pallets/providers/src/lib.rs

```diff
@@ -944,7 +951,16 @@ pub mod pallet {
         ///
         /// Emits `CapacityChanged` event when successful.
         #[pallet::call_index(6)]
-        #[pallet::weight(Weight::from_parts(10_000, 0) + T::DbWeight::get().writes(1))]
+        #[pallet::weight({
+			let weight_msp_less_deposit = T::WeightInfo::change_capacity_msp_less_deposit();
+			let weight_msp_more_deposit = T::WeightInfo::change_capacity_msp_more_deposit();
+			let weight_bsp_less_deposit = T::WeightInfo::change_capacity_bsp_less_deposit();
+			let weight_bsp_more_deposit = T::WeightInfo::change_capacity_bsp_more_deposit();
+			weight_msp_less_deposit
+				.max(weight_msp_more_deposit)
+				.max(weight_bsp_less_deposit)
+				.max(weight_bsp_more_deposit)
```

Comment:
Is this a glitch in the matrix?

Reply:
Lol do you mean the formatting? Not sure what `cargo fmt` did here